pkgs/apps/gabedit/default.nix

```
@@ -0,0 +1,67 @@
+{ lib, stdenv, fetchurl, gtk2, pkg-config, xorg, libGL, libGLU, gnome2, pango }:
+
+stdenv.mkDerivation rec {
+  pname = "Gabedit";
+  version = "2.5.1";
+
+  src = with lib.versions; fetchurl {
+    url = "mirror://sourceforge/project/gabedit/gabedit/${pname}${major version}${minor version}${patch version}/${pname}Src${major version}${minor version}${patch version}.tar.gz";
+    hash = "sha256-78sAFRrzg/Zi1TWno2orDtLxTEIIYaKIB/6qnpOL/54=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs = [
+    gtk2
+    pango
+    libGL
+    libGLU
+    xorg.libX11
+    gnome2.gtkglext
+  ];
+
+  # Adapted from platforms/CONFIG.linux64
+  configurePhase = ''
+    cat > CONFIG << EOF
+    GTKLIB = `pkg-config gtk+-2.0 --libs`  -Wl,--export-dynamic -lgtkglext-x11-1.0 -lgdkglext-x11-1.0 -lGLU -lGL -lgtk-x11-2.0 -lX11 -lgdk-x11-2.0 -latk-1.0 -lpangoft2-1.0 -lgdk_pixbuf-2.0 -lpangocairo-1.0 -lcairo -lpango-1.0 -lfreetype -lz -lfontconfig -lgobject-2.0 -lgmodule-2.0 -lglib-2.0
+    GTKCFLAGS = `pkg-config gtk+-2.0 --cflags` -I${gnome2.gtkglext}/include/gtkglext-1.0 -I${gnome2.gtkglext}/lib/gtkglext-1.0/include
```

```suggestion
    GTKCFLAGS = $(pkg-config gtk+-2.0 --cflags) -I${gnome2.gtkglext}/include/gtkglext-1.0 -I${gnome2.gtkglext}/lib/gtkglext-1.0/include
```
back quotes are not recommended anymore.